### PR TITLE
Setup list structure

### PR DESCRIPTION
**Pull request by tacettinkustu**
In this project, I built a simple HTML list of To Do tasks. The list was styled according to the specifications listed. This simple web page was built using webpack and served by a webpack dev server. The specifications are:

1. Set up a new project with webpack that is based on the webpack exercise you have already completed.
2. Create an index.html file and write your HTML markup here. 
3. Create an index.js file and set an array of some simple to do tasks (array of objects). 
4. Write a function to iterate over the tasks array and populate an HTML list item element for each task.
5. On page load render the dynamically created list of tasks in the dedicated placeholder. 
6. Create a style.css and set rules for the To Do List.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,39 @@
...
+Adding a new item.
+Removing a selected item.
+Reordering a selected item (as drag-and-drop).
+Marking a selected item as complete.
+Removing all items marked as complete at once.
+
+## Getting Started
+
+To get a local copy up and running usual steps.
+
```
Comment (ZawadiSumba66) [inline README.md:29]: - [ ] Please add instructions on how to get the project running in a local environment. Kindly refer to this[ repo](https://github.com/ZawadiSumba66/restaurant-js) and checkout how to implement the getting started part.

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
#### Hi @tacettinkustu 👋 👋 , 

Good job so far on this project 👏 👏  
However, there are some issues that you still need to work on to go to the next step but you are almost there!

## Required Changes ♻️

- [ ] Your project currently lacks a dist folder that should contain files generated by webpack after bundling up. The bundled file should then be included in a `script` tag in your `index.html` file.

Check out the screenshot below from the project requirements. 👇 

image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (tacettinkustu) [reply README.md]: I added getting started section to the readme

**Review by Temmarie — APPROVED** [review]
## Status: Approved ✔️ 
### Hi @tacettinkustu  , 
#### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
#### Congratulations! 🎉 

#### Cheers and Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Pull request merged**